src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -363,25 +391,6 @@ protected function prepare_product_price_response( \WC_Product $product, $tax_di
 		return $prices;
 	}
 
-	/**
-	 * Returns the remaining stock for a product if it has stock.
-	 *
-	 * This also factors in draft orders.
-	 *
-	 * @param \WC_Product $product Product instance.
-	 * @return integer|null
-	 */
-	protected function get_remaining_stock( \WC_Product $product ) {
-		if ( is_null( $product->get_stock_quantity() ) ) {
-			return null;
-		}
-
-		$reserve_stock  = new ReserveStock();
-		$reserved_stock = $reserve_stock->get_reserved_stock( $product, $this->get_draft_order_id() );
-
-		return $product->get_stock_quantity() - $reserved_stock;
-	}
-
```

Comment:
Was this just sitting here not used?

Reply:
It was used, but some other code moved and this moved with it.